packages/mattermost-redux/src/actions/users_queries.ts

```diff
@@ -114,17 +121,6 @@ export function transformToRecievedMeReducerPayload(user: Partial<MyDataQueryRes
         ...user,
         position: user?.position ?? '',
         roles: convertRolesNamesArrayToString(user?.roles ?? []),
-
-        // below fields arent included in the response but not inside of user rest api types
-        auth_data: '',
-        email_verified: true,
-        allow_marketing: true,
-        last_activity_at: 0,
-        bot_description: '',
-        bot_last_icon_update: 0,
-        terms_of_service_id: '',
-        terms_of_service_create_at: 0,
-        remote_id: '',
```

Comment:
Fields such as auth_dat, email_verified, allow_marketing, bot_last_icon_update are no used in the webapp, hence removing them from the query. They are available in the gql, if anyone would like to add those in the future

Reply:
Could you remove those from the type definition for `User` on the client then? Let's make sure people don't start relying on them by mistake

Reply:
You'll also probably need to remove them from the TestHelper method that generates a fake user object and update the snapshots that use that method as well too then

Reply:
Cross checked all unnecessary types from `User` and removed the ones not used.